meta/runtime.yml

```
@@ -58,6 +58,12 @@ action_groups:
   - ovirt_vmpool_facts
 
 plugin_routing:
+  lookup:
+    conjur_variable:
+      redirect: cyberark.conjur.conjur_variable
+      deprecation:
+        removal_version: 3.0.0
```

Sorry, one more:
```suggestion
        removal_version: 2.0.0
```
We discussed this on IRC and decided to remove the leftovers earlier, since this should affect only pre-1.0.0 users.